Changes between KDE 3.5.10 tag and KDE 3.5 branch r986745.
Only looking at subdirectory kdepim/libkdenetwork here.
Logs since r849627 of the branch but this might be wrong.

r986745 | winterz
provide an interface called initializeLibrary() for gpgme_check_version().
  M gpgmepp/Makefile.am
  M gpgmepp/context.cpp
  M gpgmepp/context.h


Index: gpgmepp/context.cpp
===================================================================
--- kdepim/libkdenetwork/gpgmepp/context.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdepim/libkdenetwork/gpgmepp/context.cpp	 (.../branches/KDE/3.5)	(revision 986745)
@@ -53,6 +53,10 @@
 
 namespace GpgME {
 
+  void initializeLibrary() {
+    gpgme_check_version( 0 );
+  }
+
   const char * Error::source() const {
     return gpgme_strsource( (gpgme_error_t)mErr );
   }
Index: gpgmepp/Makefile.am
===================================================================
--- kdepim/libkdenetwork/gpgmepp/Makefile.am	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdepim/libkdenetwork/gpgmepp/Makefile.am	 (.../branches/KDE/3.5)	(revision 986745)
@@ -46,7 +46,7 @@
 #   (Interfaces added/removed/changed:  CURRENT++, REVISION=0)
 #   (Interfaces added:                  AGE++)
 #   (Interfaces removed/changed:        AGE=0)
-libgpgme___la_LDFLAGS = -no-undefined -version-info 3:0:3
+libgpgme___la_LDFLAGS = -no-undefined -version-info 4:0:4
 libgpgme___la_LIBADD = $(GPGME_LIBS)
 libgpgme___la_DEPENDENCIES = $(GPGME_LIBS_DEP)
 
Index: gpgmepp/context.h
===================================================================
--- kdepim/libkdenetwork/gpgmepp/context.h	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdepim/libkdenetwork/gpgmepp/context.h	 (.../branches/KDE/3.5)	(revision 986745)
@@ -287,6 +287,8 @@
   //
   //
 
+  KDE_EXPORT void initializeLibrary();
+
   KDE_EXPORT GpgME::Error setDefaultLocale( int category, const char * value );
 
   KDE_EXPORT Context * wait( GpgME::Error & e, bool hang=true );
